member.go

```diff
@@ -22,11 +22,14 @@ import (
 )
 
 type Member struct {
-	client                   *Client
-	Id                       string   `json:"id"`
-	AvatarHash               string   `json:"avatarHash"`
-	Bio                      string   `json:"bio"`
-	BioData                  string   `json:"bioData"`
+	client     *Client
+	Id         string `json:"id"`
+	AvatarHash string `json:"avatarHash"`
+	Bio        string `json:"bio"`
+	BioData    struct {
+		Emoji struct {
+		} `json:"emoji"`
+	} `json:"bioData"`
```

Comment:
I'm using the [vim-go](https://github.com/fatih/vim-go) plugin, which automatically runs `gofmt` on save.

I got the `goimports` tool and copied the command you posted above, but it didn't change the file. Am I missing something here?